packages/@aws-cdk/aws-dynamodb-global/test/test.lambda.handler.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+const handler = require('../lib/lambda/handler');
```

Comment:
Not enough.. You'll have to mock stuff..

Reply:
lol I know, just checked in what I had gotten done so far this morning :)